hack/main.go.tmpl

```diff
@@ -71,11 +75,13 @@ func main() {
 		skipDefaultTags         = app.Flag("skip-default-tags", "Do not set default crossplane tags for resources").Bool()
 		webhookPort             = app.Flag("webhook-port", "The port the webhook listens on").Default("9443").Envar("WEBHOOK_PORT").Int()
 		metricsBindAddress      = app.Flag("metrics-bind-address", "The address the metrics server listens on").Default(":8080").Envar("METRICS_BIND_ADDRESS").String()
+		changelogsSocketPath    = app.Flag("changelogs-socket-path", "Path for changelogs socket (if enabled)").Default("/var/run/changelogs/changelogs.sock").Envar("CHANGELOGS_SOCKET_PATH").String()
 
 		namespace                  = app.Flag("namespace", "Namespace used to set as default scope in default secret store config.").Default("crossplane-system").Envar("POD_NAMESPACE").String()
 		enableExternalSecretStores = app.Flag("enable-external-secret-stores", "Enable support for ExternalSecretStores.").Default("false").Envar("ENABLE_EXTERNAL_SECRET_STORES").Bool()
 		essTLSCertsPath            = app.Flag("ess-tls-cert-dir", "Path of ESS TLS certificates.").Envar("ESS_TLS_CERTS_DIR").String()
 		enableManagementPolicies   = app.Flag("enable-management-policies", "Enable support for Management Policies.").Default("true").Envar("ENABLE_MANAGEMENT_POLICIES").Bool()
+		enableChangeLogs           = app.Flag("enable-changelogs", "Enable support for capturing change logs during reconciliation.").Default("false").Envar("ENABLE_CHANGE_LOGS").Bool()
```

Comment:
nit: We may consider making the `CHANGELOGS_SOCKET_PATH` and `ENABLE_CHANGE_LOGS` env. variables consistent in how they name the feature. We already have consistency in `changelogs-socket-path` and `enable-changelogs`. 

In my opinion consistency across the providers is also good for UX and looks like what's proposed here is already consistent with provider-kubernetes. So if we change the env. variable from `ENABLE_CHANGE_LOGS` to `ENABLE_CHANGELOGS`, then we had better rename it in provider-kubernetes also. Not sure if it's worth the hassle though.

Reply:
thanks for bringing this up - good eye! yes i think i'd like to keep it consistent amongst the providers, meaning keep what we have already shipped in existing providers like [provider-kubernetes](https://github.com/crossplane-contrib/provider-kubernetes/blob/main/cmd/provider/main.go#L76) and [provider-helm](https://github.com/crossplane-contrib/provider-helm/blob/main/cmd/provider/main.go#L64), even though it's not 100% consistent. Not worth the disruption at this point 😞